stdlib/@python2/distutils/config.pyi

```diff
@@ -13,5 +13,3 @@ class PyPIRCCommand(Command):
     boolean_options: ClassVar[list[str]]
     def initialize_options(self) -> None: ...
     def finalize_options(self) -> None: ...
-    @abstractmethod
```

Comment:
I've also removed several redefinitions of `run` abstract method. It is defined in `Command`